test/pytest/test_model_archiver.py

```diff
@@ -6,7 +6,7 @@
 import json
 import test_utils
 
-MODEL_SFILE_NAME = 'resnet18-5c106cde.pth'
```

Comment:
This feels like it's gonna break again as soon they release updated pretrained models. Is there a convention the team uses where the names increase monotonically so you can justs pick the latest available models automatically?

Reply:
@msaroufim I am not aware of any naming convention as per I know, but in general this won't break soon, as map_location now is in full support.